### ISSUE TITLE
Spurious new warning in openTypeFeatures() when turning an available feature off.

### ISSUE THREAD

**Issue by justvanrossum** [typemytype/drawbot#117]
```python
font("ACaslonPro-Regular")
openTypeFeatures(liga=False)
fontSize(80)
text("Fooooffafiaflfff", (100, 100))
```
```
*** DrawBot warning: OpenType feature 'liga_off' not available for 'ACaslonPro-Regular' ***
```

**Comment by typemytype** [comment]
oh fixed: 142a044f64beb31f00d15dab9b2e82cd3262cba9

**Comment by justvanrossum** [comment]
Thanks!
